openlcs/libs/tests.py

```diff
@@ -272,57 +272,103 @@ def setUp(self):
+                'version': '2.28',
+                'release': '151.el8',
+                'arch': 's390x',
+                'license': '',
+                'synced': True
+            },
+            {
+                'uuid': 'fe7f82c1-db81-4045-84d6-c81a9da8d146',
+                'type': 'RPM',
+                'name': 'openssl-libs',
+                'version': '1.1.1g',
+                'release': '15.el8_3',
+                'arch': 's390x',
+                'license': '',
+                'synced': True
+            },
+            {
+                'uuid': 'fe7f82c1-db81-4045-84d6-c81a9da8d147',
+                'type': 'NPM',
+                'name': '@jest/fake-timers',
+                'version': '27.2.0',
+                'release': '',
+                'arch': '',
+                'license': '',
+                'synced': True
+            },
+            {
                 'uuid': 'bb7e0e10-0a68-4bae-a490-3ff491cb1b78',
                 'type': 'CONTAINER_IMAGE',
-                'nvr': 'grc-ui-api-container-13-v2.4.0',
                 'name': "grc-ui-api-container",
                 'version': '13',
                 'release': 'v2.4.0',
                 'arch': 's390x',
-                'license': ''
-            },
-            'components': [
+                'license': '',
+                'synced': True
+            }
+        ]
+        self.group_components_data = {
+            'RPM': [
                 {
                     'uuid': 'fe7f82c1-db81-4045-84d6-c81a9da8d145',
                     'type': 'RPM',
-                    'nvr': 'glibc-minimal-langpack-2.28-151.el8',
                     'name': 'glibc-minimal-langpack',
                     'version': '2.28',
                     'release': '151.el8',
                     'arch': 's390x',
-                    'license': ""
+                    'license': '',
+                    'synced': True
                 },
                 {
                     'uuid': 'fe7f82c1-db81-4045-84d6-c81a9da8d146',
                     'type': 'RPM',
-                    'nvr': 'openssl-libs-1.1.1g-15.el8_3',
                     'name': 'openssl-libs',
                     'version': '1.1.1g',
                     'release': '15.el8_3',
                     'arch': 's390x',
-                    'license': ''
-                },
+                    'license': '',
+                    'synced': True
+                }
+            ],
+            'NPM': [
                 {
                     'uuid': 'fe7f82c1-db81-4045-84d6-c81a9da8d147',
                     'type': 'NPM',
-                    'nvr': '@jest/fake-timers-27.2.0',
                     'name': '@jest/fake-timers',
                     'version': '27.2.0',
                     'release': '',
                     'arch': '',
-                    'license': ''
+                    'license': '',
+                    'synced': True
+                }
+            ],
+            'CONTAINER_IMAGE': [
```

Comment:
It's not unified with the structure of components in sc_handler. In sc_handler, the components of 'CONTAINER_IMAGE'  should be 'CONTAINER_IMAGE': {}, not [{}]. If like to use this structure, suggest updating get_container_component() in sc_handler.

Reply:
Updated, use a uniform format